Library/Formula/mu.rb

```diff
@@ -63,4 +66,33 @@ def caveats; <<-EOS.undent
       mu index --rebuild
     EOS
   end
+
+  test do
+    mkdir (testpath/"cur")
+
+    (testpath/"cur/1234567890.11111_1.host1!2,S").write <<-EOS.undent
+      From: "Road Runner" <fasterthanyou@example.com>
+      To: "Wile E. Coyote" <wile@example.com>
+      Date: Mon, 4 Aug 2008 11:40:49 +0200
+      Message-id: <1111111111@example.com>
+
+      Beep beep!
+    EOS
+
+    (testpath/"cur/0987654321.22222_2.host2!2,S").write <<-EOS.undent
+      From: "Wile E. Coyote" <wile@example.com>
+      To: "Road Runner" <fasterthanyou@example.com>
+      Date: Mon, 4 Aug 2008 12:40:49 +0200
+      Message-id: <2222222222@example.com>
+      References: <1111111111@example.com>
+
+      This used to happen outdoors. It was more fun then.
+    EOS
+
+    system "#{bin}/mu index --muhome #{testpath} --maildir=#{testpath}"
+
+    assert_equal "1", `#{bin}/mu find --muhome #{testpath} msgid:2222222222@example.com | wc -l`.strip
```

Comment:
Something like `assert_equal 1, shell_output("#{bin}/mu ...").lines.size` would be preferable, thanks!